tests/nn/data_parallel/test_fsdp_state_dict.py

```diff
@@ -244,5 +244,28 @@ def _test_state_dict_device(self, config, rank, group, pure_fp16=False, **model_
                 assert v.dtype == expected_dtype, f"{v.dtype} != {expected_dtype}"
 
 
+@skip_if_cuda
```

Comment:
why skip_if_cuda?

Reply:
our cuda test is pretty long already. I hope to save a bit time since this test is covering a code path that's common on cpu/gpu anyway.